rules/framework.bzl

```diff
@@ -128,6 +128,10 @@ def _validate_deps_minimum_os_version(framework_name, minimum_os_version, deps):
     deps: the deps of this framework
     """
 
+    # TODO @cshi we need a number of fixes to enable this
+    if True:
```

Comment:
Can we file a GH issue to address this?

Reply:
The root of why it's broken in the mono-repo - so it might be tracked elsewhere - cc @congt